bindings/pyroot/cppyy/CPyCppyy/src/SignalTryCatch.h

```diff
@@ -71,6 +71,11 @@ using CppyyExceptionContext_t = ExceptionContext_t;
         gException = R__old; \
     }
 
-CPYCPPYY_IMPORT CppyyExceptionContext_t *gException;
+// extern, defined in ROOT Core
```

Comment:
Where exactly is this defined in ROOT?

Reply:
Ah! I thought you meant the `CppyyExceptionContext_t` type was defined in ROOT core, instead you meant the symbol, all good thanks!